### DISCUSSION TITLE
Ui Components

### DISCUSSION THREAD

**Discussion by iammohan01** [supabase/supabase#19445] [Questions] [answered]
Is it allowed To use Supabase Ui components for my personal use ??

**Comment by iammohan01** [comment]
Is it possible to use ??

**Comment by GaryAustin1** [comment, marked as answer]
Not sure what components you are referring to but the Supabase repositories are open source and you can use them for personal or business use in general.
You should always check github licenses though.
![image](https://github.com/supabase/supabase/assets/54564956/060c5f16-9cad-435c-b701-a13ff7b138fa)

**Reply by iammohan01** [reply]
Thanks 👍
